src/applications/disability-benefits/disability-rating-calculator/components/DisabilityRatingCalculator.jsx

```diff
@@ -44,21 +42,40 @@ export default class DisabilityRatingCalculator extends React.Component {
   focusLastRatingInput = () =>
     this.ratingInputRefs[this.ratingInputRefs.length - 1].focus();
 
+  focusFirstInvalidInput = () => {
+    const firstInvalidDisability = this.state.disabilities.findIndex(
+      d => d.hasError,
+    );
+    this.ratingInputRefs[firstInvalidDisability].focus();
+  };
```

Comment:
I don't think this is a super amazing way to handle focus, but it follows the pattern that's currently implemented. It might be worth revisiting.